### ISSUE TITLE
Cannot build wheel

### ISSUE THREAD

**Issue by ezatterin** [silx-kit/jupyterlab-h5web#10]
Installation fails with:

```
ERROR: Could not build wheels for jupyterlab-h5web which use PEP 517 and cannot be installed directly
```

While a potential workaround could have been:

```bash
pip install --no-use-pep517 jupyterlab_h5web
```

this also fails with:

```
ERROR: Disabling PEP 517 processing is invalid: project specifies a build backend of 
setuptools.build_meta in pyproject.toml
```

Ideas? Thanks!

**Comment by loichuder** [comment]
Hello @ezatterin ,

Thanks for checking out the extension !

I included built wheels in the latest release (0.0.3). Can you try to install this version to see if it fixes the problem ?

**Comment by ezatterin** [comment]
> Hello @ezatterin ,
> 
> Thanks for checking out the extension !
> 
> I included built wheels in the latest release (0.0.3). Can you try to install this version to see if it fixes the problem ?

Fixes on Windows 10 build 19042 running Python 3.9.1 and JupyterLab 2.2.9.

Merci!
